lib/ui.js

```diff
@@ -2,18 +2,37 @@
 const path = require('path')
 const serve = require('serve-static')
 
-module.exports.serveRedoc = function serveRedoc (documentUrl) {
+module.exports.serveRedoc = function serveRedoc (documentUrl, opts) {
+  const toKebabCase = (string) =>
+    string
+      .replace(/([a-z])([A-Z])/g, '$1-$2')
+      .replace(/[\s_]+/g, '-')
+      .toLowerCase()
+  const options = JSON.parse(JSON.stringify(opts))
+  delete options.coerce
+  delete options.htmlui
+  delete options.basePath
+
   return [serve(path.resolve(require.resolve('redoc'), '..')), function renderRedocHtml (req, res) {
     res.type('html').send(renderHtmlPage('ReDoc', `
       <link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,700|Roboto:300,400,700" rel="stylesheet">
     `, `
-      <redoc spec-url="${documentUrl}"></redoc>
+      <redoc spec-url="${documentUrl}" ${Object.keys(options).map((e) => options[e] === true ? toKebabCase(e) : '').join(' ')}></redoc>
       <script src="./redoc.standalone.js"></script>
     `))
   }]
 }
 
-module.exports.serveSwaggerUI = function serveSwaggerUI (documentUrl) {
+module.exports.serveSwaggerUI = function serveSwaggerUI (documentUrl, opts) {
+  const options = {
+    url: documentUrl,
+    dom_id: '#swagger-ui',
+    ...opts
+  }
+  delete options.coerce
```

Comment:
Similar to the above comment. We should pluck off the keys before making an object with them just to delete.

Reply:
addressed in this commit: https://github.com/wesleytodd/express-openapi/pull/47/commits/a36605409f61a1fae4e9d6507837cbd5ba9f8884